chingu-voyages/v52-tier3-team-34
# feat(client): update app metadata and add favicon

# Pull Request: Update App Metadata and Add Favicon

---

## **Description:**  
This PR enhances the branding and metadata of the React application by:  

1. **Updating `index.html`:**  
   - Changed the page title to: **"LiveScape - Discover Live Events"**.  
   - Added a meta description: *"Connecting people to the heartbeat of live music. Anytime! Anywhere!"*.  
   - Included a link to an SVG favicon for modern browsers: `/favicon.svg`.  
   - Added a `theme-color` meta tag (`#1C1C1C`) for consistent browser styling.  

2. **Adding an SVG favicon:**  
   - A new `favicon.svg` was created and added to the `client` directory to visually represent the app.

---

## **Why is this needed?**
- Improves user experience with meaningful metadata for search engines and social platforms.  
- Provides a consistent visual identity via the favicon and theme-color.  

---

## **Testing Steps:**
1. Run the app locally (`npm run dev`) and open it in a browser.  
2. Verify that the title displays as **"LiveScape - Discover Live Events"**.  
3. Check the tab for the SVG favicon.  
4. Inspect the page source to confirm the presence of updated metadata.

---

## **Screenshot:**  
![Képernyőkép 2024-12-02 173345](https://github.com/user-attachments/assets/4d0caae1-8a77-4b5e-b101-2e6ac51b0ef3)


---

## **Checklist:**  
- [x] Title updated to reflect branding.  
- [x] Meta description added.  
- [x] Favicon displays correctly in supported browsers.  
- [x] `theme-color` matches branding guidelines.
